quickOrder/database.py

```diff
@@ -0,0 +1,56 @@
+from .serializers import InventoryItemSerializer
```

Comment:
I am sorry to ask you again. Please change (refactor option using IDE) file name to database_abstraction.py